src/SFA.DAS.AssessorService.Web/ViewModels/Apply/StandardVersionViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using SFA.DAS.AssessorService.Api.Types.Models.Standards;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.AssessorService.Web.ViewModels.Apply
+{
+    public class StandardVersionViewModel
+    {
+        public Guid Id { get; set; }
+
+        public string StandardToFind { get; set; }
+
+        public string StandardCode { get; set; }
```

Comment:
I think `StandardCode` should be renamed `StandardReference` here.

In the StandardVersion model, we try to separate by LarsCode being the integer value, and IFateReference being the ST00ZZ value. 

In existing code bases they were distinguished via StandardCode = LarsCode and StandardReference = IFateReference. So if we're sticking with existing terminology, StandardRef is correct here ( as i believe it gets populate with an ST value ? )

Reply:
Changed to StandardReference